src/docs.rs

```diff
@@ -253,6 +263,7 @@ fn markdown_documentation(doc: &Option<String>) -> String {
     }
 }
 
+#[allow(clippy::integer_division)]
```

Comment:
What's the recommended way to do this without adding this attribute? I would prefer not to add these exception attributes

Reply:
Let me fix it... That was before I implemented the truncate!() macro